sklearn/feature_selection/_univariate_selection.py

```diff
@@ -230,19 +230,17 @@ def chi2(X, y):
 
 
 @_deprecate_positional_args
-def f_regression(X, y, *, center=True):
-    """Univariate linear regression tests.
+def r_regression(X, y, *, center=True):
+    """Compute Pearson R correlation coefficients between the features and
+    the target
 
```

Comment:
```suggestion

    .. versionadded:: 1.0

```

Reply:
Good catch! 

The indentation looks correct. I've looked at others contributions adding new interfaces for the 1.0: they used the `versionadded` at different places in the docstrings and some used `1.0.0` for the version number.

What should I use exactly?

Reply:
I think 1.0 is good: I can't recall versionadded directives for bug fix versions...